### PR TITLE
Update readme: usage and development sections

### PR DESCRIPTION
**Pull request by ppcano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
 
-Then run new k6 binary with the following command for using the default configuration (e.g. remote write server url set to `http://localhost:9090/api/v1/write`):
-```
-./k6 run -o xk6-prometheus-rw script.js 
+### Development
+
+For developing or testing this extension, you can build a k6 binary with the local extension using [xk6](https://github.com/grafana/xk6) with the following steps:
+
+```bash
+cd xk6-output-prometheus-remote
```
Comment (codebien) [inline README.md:19]:
```suggestion
```
It seems pretty obvious

File: README.md [hunk 2]
```diff
@@ -1,30 +1,29 @@
 # xk6-output-prometheus-remote
 
-[k6](https://github.com/grafana/k6) extension for publishing test-run metrics to Prometheus via Remote Write endpoint.
+This extension allows publishing test-run metrics to Prometheus via Remote Write endpoint.
```
Comment (codebien) [inline README.md:3]:
```suggestion
Output extension that allows publishing test-run metrics to Prometheus via Remote Write endpoint.
```

I don't like starting with `this` but maybe @heitortsergent has a better suggestion,

**Review by oleiade — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,30 +1,28 @@
...
-Then run new k6 binary with the following command for using the default configuration (e.g. remote write server url set to `http://localhost:9090/api/v1/write`):
-```
-./k6 run -o xk6-prometheus-rw script.js 
+### Development
+
+For developing or testing this extension, you can build a k6 binary with the local extension using [xk6](https://github.com/grafana/xk6) with the following steps:
+
+```bash
+
+xk6 build --with github.com/grafana/xk6-output-prometheus-remote=.
```
Comment (heitortsergent) [inline README.md:20]: Is this command building a k6 binary using the files from the main branch in this repo? If yes, would it make sense to update this to say something along the lines of "clone this repository, make changes, and run this command to build a new binary:", and then update the bash example?

File: README.md (hunk 2 above)
Comment (heitortsergent) [reply README.md:3]: I think I like the original version better tbh... But an alternative if you don't wanna start with `this`, and I agree with that feedback:

"The xk6-output-prometheus-remote extension allows you to publish test-run metrics to Prometheus via Remote Write endpoint."
(repetition isn't always bad)

Another feedback, how about including a link to the [k6 extensions page](https://k6.io/docs/extensions/)? If a user lands in this repository and doesn't know much about k6 extensions, I think that might give them the best explanation of what this is.

**Review by heitortsergent — COMMENTED** [review]
Just a couple of comments, but no blockers. :)

**Review by codebien — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ppcano) [reply README.md]: https://github.com/grafana/xk6-output-prometheus-remote/pull/151/commits/21a830d8f73b1cc6f571e96226ed2cd9b9355bc1

File: README.md (hunk 3 above)
Comment (ppcano) [reply README.md]: This is an advanced case, and requires a few more instructions and context. Then, we either document the entire process (which seems out of the scope) or refer to the primary guides.

In this case:
- shows the core command and try to highlight the simplicity
- refer to the docs

**Pull request merged**